src/index.rst

```diff
@@ -3,6 +3,31 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
+conda-forge documentation
+=========================
+
+What is conda-forge?
+--------------------
+
+Conda-forge is a community effort that provides conda packages for a wide range of software.
+
+**Missing a package that you would love to install with conda?**
+
+Chances are we have already packaged it for you. You can `search <https://anaconda.org/>`__ for packages online. Look out for packages provided by our ``conda-forge`` organization.
+
+**Cannot find a package or only outdated versions of a package?** - Everybody is welcome to contribute to our package stack!
+
+- To get started contributing packages see here :ref:`becoming_involved`.
```

Comment:
Need a comma after "packages".

Reply:
Also, the ref link needs to be ```see :ref:`here <becoming_involved>`.```

Reply:
@kathatherine, actually the link works fine without the angle brackets, and if I do add them it giving an error. I think it is how pages, which are already in the documentation are linked and brackets are used for the external links.